bundles/FileExplorerBundle/src/Installer.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Pimcore\Bundle\FileExplorerBundle;
+
+use Pimcore\Extension\Bundle\Installer\SettingsStoreAwareInstaller;
+
+class Installer extends SettingsStoreAwareInstaller
+{
+    const USER_PERMISSIONS = [
+        'fileexplorer'
+    ];
+
+    public function install()
+    {
+        $this->addPermissions();
+        parent::install();
+    }
+
+    public function uninstall()
+    {
+        $this->removePermissions();
+        parent::uninstall();
+    }
+
+    protected function addPermissions()
+    {
+        $db = \Pimcore\Db::get();
+
+        /**
+         * The following code is here for backwards compatibility reason.
+         * If the permissions are already installed, the installer should not generate
+         * any further errors and only add permissions which
+         */
```

Comment:
@brusch this is a good point. If we do not mark the bundle as installed via a migration, and someone upgrades from 10 to 11 and tries to install the file explorer bundle, but the permission is already there the installation of the bundle will fail.

Maybe we should remove this permission via a migration for all bundles that will get its own repo?

Reply:
@mattamon @aashan10 yes, we should do so! 👍

Reply:
@brusch We should update the refactoring tips once again. Should this be the standard, to remove permissions via migrations if the bundle gets its own repo?

@aashan10 Could you please create a Migration in the Core to remove the permission. I think we then do not need all the compatibility code in the installer.